multiapps-controller-process/src/test/java/org/cloudfoundry/multiapps/controller/process/steps/CreateOrUpdateAppStepTest.java

```diff
@@ -167,6 +167,17 @@ void testInjectServiceKeysCredentialsInAppEnv() {
                             .get(APP_NAME));
     }
 
+    private CloudServiceKey buildCloudServiceKey(Map<String, String> serviceKeyCredentials) {
```

Comment:
These changes in `CloudServiceKey` object reminds me that we have to test how it will be handled with old running process and update to latest version including these changes.

Reply:
Added a new commit with backwards compatible logic